lib/solarwinds_apm/patch/tag_sql/sw_dbo_utils.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module SolarWindsAPM
+  module Patch
+    module TagSql
+      module SWODboUtils
+        def self.annotate_span_and_sql(sql)
+          return sql if sql.to_s.empty?
+
+          current_span = ::OpenTelemetry::Trace.current_span
+
+          annotated_sql = ''
+          if current_span.context.trace_flags.sampled?
+            traceparent = SolarWindsAPM::Utils.traceparent_from_context(current_span.context)
+            annotated_traceparent = "/*traceparent='#{traceparent}'*/"
+            current_span.add_attributes({ 'sw.query_tag' => annotated_traceparent })
```

Comment:
hmm, java uses `QueryTag`. it looks like we need to converge to the same name? how about `sw.QueryTag`?

Reply:
Yes, Lin mentioned to me Java use QueryTag. I believe she wants sw.query_tag eventually. I can confirm with her.